### PR TITLE
[INLONG-6838][Manager] Support multi addresses for ES load node

### LINKED ISSUE #6838: [Feature] Support multi addresses for ES load node
### Description

 Support adding multi addresses for ES load node.

- before
host: host_name
port: 9092
- after
hosts: "http://host_name:9092;http://host_name:9093"

include manager and dashboard.

https://github.com/apache/inlong/pull/6817 fixed a bug for one address.

### Use case

_No response_

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wangpeix**
### Prepare a Pull Request
- Title: [INLONG-6838][Manager] Support multi addresses for ES load node

- Fixes #6838 

### Motivation

Support multi addresses for ES load node

### Modifications

Get `url` parameter from ES data node. And then, support multi addresses for ES load node.

### CODE REVIEW

**Comment by dockerzhang** [comment]
@wangpeix Please fix the failed UT, thanks.

**Review by healchow — APPROVED** [review]

File: inlong-dashboard/src/locales/cn.json [hunk 1]
```diff
@@ -388,6 +388,7 @@
   "meta.Nodes.MySQL.Url": "地址",
   "meta.Nodes.MySQL.BackupUrl": "备份地址",
   "meta.Nodes.ES.Host": "主机",
+  "meta.Nodes.ES.HostHelp": "要连接到的一台或多台Elasticsearch主机，例如 'http://host_name:9092;http://host_name:9093'",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:391]: 要连接到的一台或多台Elasticsearch主机
->
要连接到的一台或多台 Elasticsearch 主机

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sink/ElasticsearchSinkServiceTest.java [hunk 2]
```diff
@@ -97,4 +109,16 @@ public void testGetAndUpdate() {
         deleteSink(sinkId);
     }
 
+    private void prepareEsDataNode(String dataNodeName) {
+        ElasticsearchDataNodeRequest request = new ElasticsearchDataNodeRequest();
+        request.setName(dataNodeName);
+        request.setType(DataNodeType.ELASTICSEARCH);
+        request.setUrl("http://127.0.0.1:9200;http://127.0.0.1:9200");
```
Comment (dockerzhang) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sink/ElasticsearchSinkServiceTest.java:116]: http://127.0.0.1:9200;http://127.0.0.1:9200
->
http://127.0.0.1:9200;http://127.0.0.1:9300

It's better to use a different URL.

Comment (wangpeix) [reply inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sink/ElasticsearchSinkServiceTest.java:116]: Okay, I will improve it

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/es/ElasticsearchResourceOperator.java [hunk 3]
```diff
@@ -141,25 +139,7 @@ private ElasticsearchConfig getElasticsearchConfig(SinkInfo sinkInfo, Elasticsea
             config.setUsername(esInfo.getUsername());
             config.setPassword(esInfo.getPassword());
         }
-        config.setHost(esInfo.getHost());
-        config.setPort(esInfo.getPort());
-        // read from data node if not supplied by user
```
Comment (Oneal65) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/es/ElasticsearchResourceOperator.java:146]: Whether or not  to add the logic of reading from data node

**Review by gong — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/es/ElasticsearchResourceOperator.java (hunk 3 above)
Comment (wangpeix) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/es/ElasticsearchResourceOperator.java:146]: These configs used here comes from `stream_sink` table,   `stream_sink` table data comes from the data node and saved. I think may be it's ok to use directly here.

**Review by Oneal65 — APPROVED** [review]

**Pull request merged**